### ISSUE TITLE
Unexpected keyword argument 'past_detections_length'

### ISSUE THREAD

**Issue by ManuConcepBrito** [tryolabs/norfair#76]
I was trying out to increase the length of a tracked object's past detections, but got an 'Unexpected keyword argument'. I am running norfair==0.3.1 (pip installed). I think the version of the pip package doesn't correspond with the one published in github? I attach a simple script to reproduce the issue: 

```
from norfair import Tracker

max_distance_between_points=50

def euclidean_distance(detection, tracked_object):
    return np.linalg.norm(detection.points - tracked_object.estimate)

tracker = Tracker(
        distance_function=euclidean_distance,
        distance_threshold=max_distance_between_points,
        past_detections_length=50

    )
```

**Comment by joaqo** [comment]
Yeah thats correct, you'll need to clone the repo and install it locally to run the latest features. Its a pure python (no compilation steps needed) package so it should be easy.

We are working on adding documentation/examples for the new features before making an official pip release with them, sorry for the inconvenience!

**Comment by spikyknox** [comment]
Encounter the same problem, I installed it directly from the github pip install git+https://github.com/tryolabs/norfair.git@master to have the latest version and now it work. ( Ps: Thx for the new features it is really useful for us)

**Comment by joaqo** [comment]
Awesome, thats great to hear!
